src/Aggregation/Histogram.php

```diff
@@ -14,11 +14,16 @@ class Histogram extends AbstractSimpleAggregation
      * @param string     $field    the name of the field on which to perform the aggregation
      * @param int|string $interval the interval by which documents will be bucketed
      */
-    public function __construct(string $name, string $field, $interval)
+    public function __construct(string $name, string $field, $interval = null)
     {
         parent::__construct($name);
         $this->setField($field);
-        $this->setInterval($interval);
+
+        if (null !== $interval) {
+            trigger_deprecation('ruflin/elastica', '7.1.0', 'The "%s::__construct" third argument is deprecated, use "setDateInterval()" or "setCalendarInterval()" instead. It will be removed in 8.0.', __CLASS__);
```

Comment:
```suggestion
            trigger_deprecation('ruflin/elastica', '7.1.0', 'Argument 3 passed to "%s()" is deprecated, use "setDateInterval()" or "setCalendarInterval()" instead. It will be removed in 8.0.', __METHOD__);
```